import java.io.IOException;

public class SecondCode {

    //TRY, CATCH, FINALLY

        public static void main(String[] args){
            System.out.println("Start main");

            System.out.println(10/0);
            // try{
            //     System.out.println(10/0);//risky code 
            // }
            // catch(ArithmeticException e){
            //     System.out.println("Exception aaley");
            // }
            
            System.out.println("End main");
        }

       
    
}
